ntbs-service/wwwroot/css/caseManager.scss

```diff
@@ -73,8 +73,53 @@
 }
 
 .case-manager-search-button {
```

Comment:
I think it's worth adding a comment here saying that this is based on `nhsuk-search__submit` with a link to the source code

Reply:
Yep, will do!